components/Ripple/src/private/MDCRippleLayer.h

```diff
@@ -45,6 +45,11 @@ typedef void (^MDCRippleCompletionBlock)(void);
  */
 @property(nonatomic, assign) CFTimeInterval rippleTouchDownStartTime;
 
+/**
+ The radius the ripple expands to when activated.
+ */
+@property(nonatomic, assign) CGFloat rippleRadius;
```

Comment:
Put another way:  unless there is a technical reason to do otherwise I would expect both of the radius APIs to have identical names.

Reply:
I feel similarly, unless you think the old name is incorrect or this new naming is significantly better.

Reply:
This API is on `MDCRippleLayer` which is coming from #7350, this PR also adds an API to `MDCRipppleView` and I will update that. I originally had it as `maxRippleRadius` but then just thought the name was a minor improvement. It seems this could cause more headache during migration so I'll go back to `maxRippleRadius`.

Reply:
@codeman7 and I discussed offline that consistency is important here. @codeman7 feels that `maximumRadius` is the most preferred API here, and I agree. The migration cost for clients will be an additional find-and-replace, but I consider this a low cost in relation to the value of aligning the naming on Objective-C naming conventions (e.g. `minimumFontSize` in UITextField).

Reply:
SGTM, thanks!